openapiv3/OpenAPIv3.pb.go

```diff
@@ -1623,8 +1623,8 @@ type Info struct {
 	Contact                *Contact    `protobuf:"bytes,4,opt,name=contact,proto3" json:"contact,omitempty"`
 	License                *License    `protobuf:"bytes,5,opt,name=license,proto3" json:"license,omitempty"`
 	Version                string      `protobuf:"bytes,6,opt,name=version,proto3" json:"version,omitempty"`
-	SpecificationExtension []*NamedAny `protobuf:"bytes,7,rep,name=specification_extension,json=specificationExtension,proto3" json:"specification_extension,omitempty"`
-	Summary                string      `protobuf:"bytes,8,opt,name=summary,proto3" json:"summary,omitempty"`
+	Summary                string      `protobuf:"bytes,7,opt,name=summary,proto3" json:"summary,omitempty"`
+	SpecificationExtension []*NamedAny `protobuf:"bytes,8,rep,name=specification_extension,json=specificationExtension,proto3" json:"specification_extension,omitempty"`
```

Comment:
Shouldn't field ids changes be forbidden? This is a breaking api change.

Reply:
@fenollp

Are you means above?

```starlark
	SpecificationExtension []*NamedAny `protobuf:"bytes,7,rep,name=specification_extension,json=specificationExtension,proto3" json:"specification_extension,omitempty"`
	Summary                string      `protobuf:"bytes,8,opt,name=summary,proto3" json:"summary,omitempty"`
```

to

```starlark
	Summary                string      `protobuf:"bytes,7,opt,name=summary,proto3" json:"summary,omitempty"`
	SpecificationExtension []*NamedAny `protobuf:"bytes,8,rep,name=specification_extension,json=specificationExtension,proto3" json:"specification_extension,omitempty"`
```

?